### PR TITLE
Added Microsoft-proprietary msIsStereo3D property

### PR DESCRIPTION
**Pull request by mattwojo**

### CODE REVIEW

File: api/HTMLVideoElement.json [hunk 1]
```diff
@@ -610,6 +610,57 @@
             "deprecated": true
           }
         }
+      },
+       "msIsStereo3D": {
```
Comment (ExE-Boss) [inline api/HTMLVideoElement.json]: Wrong indentation

File: api/HTMLVideoElement.json [hunk 2]
```diff
@@ -610,6 +610,57 @@
...
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLVideoElement/msIsStereo3D",
+          "support": {
```
Comment (ExE-Boss) [inline api/HTMLVideoElement.json]: Sort the `support` block alphabetically

**Comment by mattwojo** [comment]
Not quite understanding the build error here. :(

File: api/HTMLVideoElement.json (hunk 2 above)
Comment (mattwojo) [reply api/HTMLVideoElement.json]: Not quite sure what you mean... seems to match the other property entries...?

Comment (connorshea) [reply api/HTMLVideoElement.json]: Generally it's supposed to be alphabetical, though it hasn't really been enforced in the past. This file has weird ordering, not sure what's up with it.

Comment (ExE-Boss) [reply api/HTMLVideoElement.json]: @mattwojo See issue #398 and pull request #1882.

File: api/HTMLVideoElement.json [hunk 3]
```diff
@@ -654,6 +652,9 @@
             "samsunginternet_android": {
               "version_added": false
             }
```
Comment (ExE-Boss) [inline api/HTMLVideoElement.json]: This is missing the `,` after the closing `}`, which is a syntax error.

File: api/HTMLVideoElement.json [hunk 4]
```diff
@@ -610,6 +610,58 @@
...
+            "edge": {
+              "version_added": true
+            },
+             "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": false
+            },
+
```
Comment (ExE-Boss) [inline api/HTMLVideoElement.json]: Extraneous empty line.

Comment (mattwojo) [reply api/HTMLVideoElement.json]: Thanks... and apologies, I'll get the hang of this sooner or later.

File: api/HTMLVideoElement.json [hunk 5]
```diff
@@ -611,6 +611,57 @@
...
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": false
+            },
```
Comment (ExE-Boss) [inline api/HTMLVideoElement.json]: This line has trailing whitespace (I really should probably add something to the error output to highlight this).

**Comment by mattwojo** [comment]
WOOO!

**Comment by ExE-Boss** [comment]
I’d recommend using [Visual Studio Code](https://code.visualstudio.com/) with the [EditorConfig for VS Code](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) extension.

**Pull request merged**